pkgs/top-level/python-packages.nix

```diff
@@ -4011,6 +4013,18 @@ in {
 
   mapbox = callPackage ../development/python-modules/mapbox { };
 
+  mapbox-earcut = callPackage ../development/python-modules/mapbox-earcut { };
+
+  mapsplotlib = callPackage ../development/python-modules/mapsplotlib { };
+
+  marionette_driver = callPackage ../development/python-modules/marionette-harness/marionette_driver.nix { };
+
+  marionette-harness = callPackage ../development/python-modules/marionette-harness { };
```

Comment:
ofBorg is complaining about this line. Was this the result of a merge?

Reply:
hmmm, yes there was a merge of master into this branch. but i don't understand where this comes from. I didn't touch marionette*

Reply:
found out i messed up the merge. now it should be okay